package.json

```diff
@@ -19,8 +19,9 @@
     "flatten": "solidity-flattener"
   },
   "dependencies": {
-    "@openzeppelin/contracts": "3.2.2-solc-0.7",
-    "solc": "^0.7.4"
+    "@ethersproject/providers": "^5.0.14",
+    "@openzeppelin/contracts": "^3.2.1-solc-0.7",
+    "solc": "0.7.2"
```

Comment:
- Was the rolling back of solc and openzeppelin versions intentional? 
- Also I think, a separate entry for `@ethersproject/providers` should not be needed. Can you try without it if you encounter any issues?
- `npm i ethers@latest` should install you latest ethers (this might need package lock file deleted).

Reply:
No , it's not intentional  if just got some errors while  using `npm install` 

So I just fix ethers to  `"ethers": "5.0.19",` version

Reply:
Duplicate of #

Reply:
I meant these packages: `solc` and `@openzeppelin/contracts`. Do you intend to have these changes?